Sources/Helpers/PrepareForSkeletonProtocol.swift

```diff
@@ -12,22 +12,50 @@ protocol PrepareForSkeleton {
     func prepareViewForSkeleton() 
 }
 
-extension UILabel: PrepareForSkeleton {
-    func prepareViewForSkeleton() {
-        text = " "
-        resignFirstResponder()
+extension UIView: PrepareForSkeleton {
+    @objc func prepareViewForSkeleton() {
+        _ = startTransitionIfAvailable()
+    }
+    fileprivate func startTransitionIfAvailable() -> Bool {
+        if let transitionType = currentSkeletonConfig?.transition {
```

Comment:
Besides, here we could use `guard` instead of if block.

```swift
guard let transitionType = currentSkeletonConfig?.transition else {
			completion?()
			return
		}
``